sale_invoice_negative_quantities/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module allows invoicing negative quantities on sales order, when product policy is set to delivered quantities.
```

Comment:
@BernatObrador This could be enhanced by adding a CONTEXT to explain when it is useful.